src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -219,9 +214,10 @@ def __init__(
         deterministic: bool = True,
         sampling_temperature: float = 1.0,
```

Comment:
Aren't these functionally contained within GenerationConfig as `do_sample` and `temperature`? 
https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig.do_sample
https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig.temperature

Reply:
happy to add this in a follow-up as we support for generation config properties